src/components/tops/actions/DeleteEntities.vue

```diff
@@ -12,16 +12,29 @@
       {{ errorText }}
     </div>
   </div>
+  <hard-delete-modal
+    :active="modals.deleteConfirmation"
+    :error-text="errorText"
+    :is-loading="isLoading"
+    :is-error="isError"
+    :text="hardDeleteTextComputed"
+    :lock-text="hardDeleteLockTextComputed"
+    @cancel="modals.deleteConfirmation = false"
+    @confirm="confirm"
+    vif="modals.deleteConfirmation"
+  />
```

Comment:
- Code typo on `v-if`
- `active` can always be set to `true` by default if a `v-if` is used.

```suggestion
  <hard-delete-modal
    active
    :error-text="errorText"
    :is-loading="isLoading"
    :is-error="isError"
    :text="hardDeleteTextComputed"
    :lock-text="hardDeleteLockTextComputed"
    @cancel="modals.deleteConfirmation = false"
    @confirm="confirm"
    v-if="modals.deleteConfirmation"
  />
```